test/NuGet.Frameworks.Test/NuGetFrameworkParseTests.cs

```diff
@@ -255,6 +255,7 @@ public void NuGetFramework_ParsePCLTest()
         [InlineData("portable-net45+wp8+win8+wpa")]
         [InlineData("portable-net45+win8+wp8+wpa81+monotouch+monoandroid")]
         [InlineData(".NETPortable,Version=v0.0,Profile=Profile259")]
+        [InlineData("portable-net45+wp8+win+wpa+win8")]
```

Comment:
This needs more tests. Vary and duplicate other frameworks. It should also check equivalent frameworks such as netcore45 instead of win8.

portable-net45+wp8+win+wpa+netcore+netcore45
portable-net450+net4.5+net45+wp8+wpa+win8+wpa81
portable-win8+net45+wp8+wpa81+win8+win8